source/administration/devops-command-center.rst

```diff
@@ -0,0 +1,268 @@
+
+Listed incidents include the following information:
+
+- The current Commander’s profile picture and username
+- Zero or more checklist items
+- The channels associated with the incident
+- A button to end the incident
+
+If the incident is active, but in a private channel, you won't be able to see the incident channel name or select **End Incident**. However, the Commander name and incident checklist is displayed. If you're a participant in an incident channel, you can modify incident details from within that channel.
+
+You can view all incidents in the incident Backstage via **Main Menu > Incidents & Playbooks** and then select an incident to view the following details:
+
+- Incident name
+- A link icon to open the corresponding incident channel
+- Status (**Ongoing** or **Ended**)
+- The Commander, including profile picture and username
+- A prompt to export the incident channel
+- The total duration
+- The number of members involved in the incident
+- The number of messages posted to the channel, including another prompt to open the corresponding incident channel
+- A graph depicting when each checklist item was completed
+
+- The **Duration** widget displays the duration of the incident. While the incident is ongoing, the end time is displayed as **Ongoing**. When the incident has ended, it shows the end time (in the user's timezone). 
+- The **Members Involved** widget indicates the total number of users that participated in the channel, either
+by posting a message, being assigned as Commander, or interacting with a checklist. This number is not affected by users leaving the channel, or users joining the channel but not participating.
+- The total number of messages displayed includes messages posted by both users and bots (including the incident response bot). It does not include system or ephemeral messages.
+
+**Filtering incidents**
+
+Incidents can be filtered by incident name, Commander, and incident status.
+
+**Browsing related channels**
+
+Incident members see a link to the incident channel at the bottom of the incident details. Clicking the channel name navigates to the incident channel. This section is not displayed when the active user is not an incident member.
+
+Exporting channels
+^^^^^^^^^^^^^^^^^
+
+If your server is licensed for E20, and the channel export plugin is installed and active, navigate to **Main Menu > Incidents & Playbooks**, select an incident, then choose **Export Incident Channel** in the top-right corner to download the contents of the incident channel as a CSV. The file excludes attachments, but includes system messages.
+
+If you have an E20 license but the channel export plugin is not installed, or the plugin is installed but not enabled, it’s not possible to select **Export Incident Channel**.
+
+To install and activate the plugin, navigate to the plugins menu and follow the steps provided.
```

Comment:
What is the "plugins menu" in this context? Wondering if we should just say "install the plugin from the private beta marketplace" for now.

Reply:
Thanks for picking this up, I'd meant to comment on this as I realized I don't know the steps to install/activate the plugin and needed to confirm with you. Are there specific steps for that, or can it be done via the Main Menu > Download Apps?

Reply:
Let's rely on / link to https://docs.mattermost.com/administration/plugins.html#plugin-marketplace to drive plugin installation.